File: src/lolicon.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class lolicon
{
	void intice() {
        	// Nothing because weebs are already attracted to underage anime girls
        	return;
    	}
    
	public static void main(String[] args) throws IOException {
	    	System.out.println("What\'s the girl\'s age?");
		BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        	int age = Integer.parseInt(reader.readLine());
		CallTheFbi(age);
	}
	
	static void CallTheFbi(int age) {
		if (age < 15) {
			try{
				ProcessBuilder processBuilder = new ProcessBuilder();
				processBuilder.command("call_911");
				processBuilder.start();
			}   
			catch (IOException e) {
	   	         	e.printStackTrace();
           	 	}
		} else {
			System.out.println("Probably just something weird happened.");
        	}
		return;
	}
}

File: src/lolicon.kt
import java.util.Scanner

fun intice() {
  // Nothing because weebs are already attracted to underage anime girls
}

fun callthefbi(age: Int) {
  if (age < 15){
    Runtime.getRuntime().exec("call_911")
  } else {
    println("Probably just something weird happened")
  }
}

fun main(args: Array<String>) {
  val reader = Scanner(System.`in`)
  println("What\'s the girl\'s age? ")
  var age:Int = reader.nextInt()
  callthefbi(age)
}
